dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -38,6 +38,8 @@ local tputests = base.BaseTest {
       export PL_RUN_TPU_TESTS=1
       cd tests/tests_pytorch
       coverage run --source=pytorch_lightning -m pytest -vv --durations=0 ./
+      test_exit_code=$?
+      test $test_exit_code -eq 0
```

Comment:
Have you tried setting

`set -e`

at the beginning?

Reply:
Confirm it works. Tested by inserting manual assert False (standalone and normal test!) Great!!

<img width="799" alt="image" src="https://user-images.githubusercontent.com/5495193/186357010-fbbceecb-71f2-420d-a644-e3189db2be73.png">

Reply:
Then this is not required

```suggestion
```

Reply:
How do you know for sure? It will take again half a day for me to trigger TPU CI to verify with a force test failure xD

Reply:
Because the screenshot you posted shows that the pytest command failed

The explicit check below `run_standalone_tests` could be removed too